crates/fluvio-benchmark/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "fluvio-benchmark"
+version = "0.0.0"
+edition = "2021"
+
```

Comment:
can you license, description, published, etc.

Reply:
Ok I think I've done this correctly but please double check.

Reply:
looks good